src/pages/Welcome/Welcome.scss

```diff
@@ -0,0 +1,47 @@
+@import "../../styles/Colors.scss";
+@import "..//../styles/Mixins.scss";
+@import "../../styles/Fonts.scss";
+
+.welcomePage {
+    height: 100vh;
```

Comment:
add it to globals.scss

Reply:
Does not work properly in global styles.